Scripts/parameters/assignment.py

```diff
@@ -230,12 +230,12 @@
 )
 assignment_classes = {
     "hw": "work",
-    "hc": "leisure",
-    "hu": "leisure",
+    "hc": "work",
+    "hu": "work",
     "hs": "leisure",
     "ho": "leisure",
-    "hoo": "work",
-    "wo": "work",
+    "hoo": "leisure",
+    "wo": "leisure",
```

Comment:
These classifications are consistent with Helmet estimation input data and fixed values-of-time, but I don't have any other sources of information to justify this.

Reply:
Yes, you are right. Maybe this change would be ok. This will however somewhat affect model results, so it is questionable whether this could really be seen as only a patch. Should we schedule for v4.1?

Reply:
Yep, it's good procedure to notify users when model results change. 

Would it be a good way to collect these types of issues to [Helmet estimation repo](https://github.com/HSLdevcom/helmet-estimation/issues)? If there is for example reference that value-of-time is lower for study / school tours and that should be considered when updating estimation.